### DISCUSSION TITLE
Won't run on windows 11

### DISCUSSION THREAD

**Discussion by xenotaph** [ElPumpo/TinyNvidiaUpdateChecker#158] [General]
I have a rather fresh install of windows 11, but I get this error:

TinyNvidiaUpdateChecker.v1.16.2.exe
You must install .NET to run this application.

App: C:\Users\fanni\Downloads\TinyNvidiaUpdateChecker.v1.16.2.exe
Architecture: x86
App host version: 7.0.2
.NET location: Not found

Learn about runtime installation:
https://aka.ms/dotnet/app-launch-failed

Download the .NET runtime:
https://aka.ms/dotnet-core-applaunch?missing_runtime=true&arch=x86&rid=win10-x86&apphost_version=7.0.2

So I installed the .net runtime, same problem, so I thought maybe I need the x64 version and installed that, still doesn't work.

What am I not getting?

**Comment by HawaiiBeach** [comment]
Hi. You did not install the .NET 7 Desktop Runtime x86 properly. Sadly the link from Microsoft is incorrect.

[Please follow the updated readme](https://github.com/ElPumpo/TinyNvidiaUpdateChecker#net-desktop-runtime-7)
